<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Each_game_history</class>
 <widget class="QWidget" name="Each_game_history">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pictures.qrc">
    <normaloff>:/new/prefix1/Skull.jpg</normaloff>:/new/prefix1/Skull.jpg</iconset>
  </property>
  <widget class="QLabel" name="label_backoround">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/each_game_history_background.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_opponent_username">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Viner Hand ITC&quot;;
color: rgb(57, 36, 29);</string>
   </property>
   <property name="text">
    <string>Opponent: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_opponent_score">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>181</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Viner Hand ITC&quot;;
color: rgb(57, 36, 29);</string>
   </property>
   <property name="text">
    <string>Opponent score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_your_score">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>130</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Viner Hand ITC&quot;;
color: rgb(57, 36, 29);</string>
   </property>
   <property name="text">
    <string>Your score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Viner Hand ITC&quot;;
color: rgb(57, 36, 29);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="opponent_username">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Viner Hand ITC&quot;;
color: rgb(57, 36, 29);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="your_score">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Viner Hand ITC&quot;;
color: rgb(57, 36, 29);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="opponent_score">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Viner Hand ITC&quot;;
color: rgb(57, 36, 29);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_result">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>131</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Viner Hand ITC&quot;;
color: rgb(57, 36, 29);
</string>
   </property>
   <property name="text">
    <string>Result:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_screenshot">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_frame">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/pic_frame.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_backoround</zorder>
  <zorder>label_opponent_username</zorder>
  <zorder>label_opponent_score</zorder>
  <zorder>label_your_score</zorder>
  <zorder>result</zorder>
  <zorder>opponent_username</zorder>
  <zorder>your_score</zorder>
  <zorder>opponent_score</zorder>
  <zorder>label_result</zorder>
  <zorder>pushButton_back</zorder>
  <zorder>label_frame</zorder>
  <zorder>label_screenshot</zorder>
 </widget>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
